modules/IndexRedirect.js

```diff
@@ -41,4 +21,24 @@ class IndexRedirect extends Component {
 
 }
 
+IndexRedirect.propTypes = {
+  to: string.isRequired,
+  query: object,
+  state: object,
+  onEnter: falsy,
+  children: falsy
+}
+IndexRedirect.createRouteFromReactElement = function (element, parentRoute) {
```

Comment:
Sure, but what's off exactly? I'm adding a space between the propTypes and the function creation, is that what you're talkin about? Just want to make sure I understand.